src/_assets/javascripts/components/moving-allowances-table.js

```diff
@@ -6,13 +6,27 @@
 
     $window.on('update', this.events.update.bind(this));
 
+    this.$el.find('.view-details').each(function() {
+      var $clone = $(this.hash).clone();
+
+      $clone.find('h4, .return-to-moving-allowances').remove();
+      $clone.removeAttr('id').insertAfter($(this));
+    });
+
+    this.$el.addClass('moving-allowances-table-enhanced');
+
     this.toggle(true);
   };
 
   MovingAllowancesTable.prototype = {
     events: {
       update: function(event, data) {
         this.toggle(data.payGrade !== this.$el.data('payGrade'));
+
+        this.$el.attr({
+          'data-dependency-status': data.dependencyStatus,
+          'data-move-type': data.moveType
+        });
```

Comment:
Adding `data-*` attributes to the `<table>` using the user-provided values as styling hooks for content with the `<table>`.